src/NewRecipe.css

```diff
@@ -0,0 +1,65 @@
+.New {
+  margin-bottom: 50px auto;
+  margin-top: 0px auto;
+  background-color: #282c34;
+  min-height: 100vh;
+  clear: both;
+  padding: 0;
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+}
+
+.New-Body {
+  display: flex;
+  width: 500px;
+  flex-direction: column;
+  align-items: center;
+  margin-top: 25px;
+}
+
+.New-Parameters {
+  display: flex;
+  width: 100%;
+  flex-direction: column;
+  margin-top: 15px;
+  color: white;
+}
+
+.New-Submit {
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  margin-top: 25px;
+  border-top: solid lightgray;
+  padding: 25px;
+}
+
+.New-InputColor {
+  background-color: rgb(200, 200, 200);
+}
+
+.New-TextColor {
+  color: rgb(200, 200, 200);
+}
+
+.ant-rate-star.ant-rate-star-zero {
+  accent-color: black;
+}
+
+.ant-input-group-addon {
+  background-color: darkgray;
+  border: 0px;
```

Comment:
does the border exist here? Is the 0px value necessary?

Reply:
![image](https://user-images.githubusercontent.com/25393502/180329064-cc4e9eb8-694e-469b-b2c8-a5297bd8a1bc.png)
 the border indeed exists

![image](https://user-images.githubusercontent.com/25393502/180329085-dab33e1a-90f9-42b1-81a6-856f8ee0684d.png)